### DISCUSSION TITLE
Compatibility with Gatsby

### DISCUSSION THREAD

**Discussion by danequin** [harness-software/wp-graphql-gravity-forms#77] [General]
Hi,

Gatsby reserves the field "fields". Is it easy enough to rename this (e.g. formFields) when registering the form to GraphQL? If so, if anyone can point me in the right direction that would be much appreciated 🙂

Thanks

**Comment by justlevine** [comment]
This is a known issue with the new `gatsby-source-wordpress`, and sadly it's not as simple as simply renaming the property to something other than `field`, since Union types in general are broken.

In the interim, you apply the following PRs to your local copy of `gatsby-source-wordpress`:
https://github.com/gatsbyjs/gatsby/pull/29717
https://github.com/gatsbyjs/gatsby/pull/29751

**Comment by danequin** [comment]
Thanks Dovid, got it working 👍

**Comment by justlevine** [comment]
v0.5.0 is `gatsby-source-wordpress` compatible.

**Comment by danequin** [comment]
Awesome, thanks for the update 😀
